.github/workflows/tests_with_database.yml

```diff
@@ -7,6 +7,67 @@ on:
   pull_request: {}
 
 jobs:
+  mysql:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - mysql:5.7
+          - mysql:8
+          - mysql:latest
+          - mariadb:10.2
+          - mariadb:10.3
+          - mariadb:10.4
+          - mariadb:10.5
+          - mariadb:10.6
+          - mariadb:10.7
+          - mariadb:latest
```

Comment:
That version list looks outdated. For MySQL, both 8.0 and 8.4 are LTS versions (see https://en.wikipedia.org/wiki/MySQL#Release_history). Similarly, at least MariaDB 10.11, 11.1, and 11.2 should be added (see https://en.wikipedia.org/wiki/MariaDB#Versioning).